packages/core/src/codewhispererChat/controllers/chat/controller.ts

```diff
@@ -915,8 +912,13 @@ export class ChatController {
             return []
         }
         const workspaceFolder = contextCommands[0].workspaceFolder
-        const prompts = await LspClient.instance.getContextCommandPrompt(contextCommands)
-        if (prompts.length === 0) {
+        let prompts = []
+        try {
+            prompts = await LspClient.instance.getContextCommandPrompt(contextCommands)
+        } catch {
+            return []
```

Comment:
do you need to return here? Maybe just continue executing below code.

Reply:
if there is any exception or empty list, triggerPayload.additionalContents = [] at line 926 still needs to be executed.